packages/medusa-plugin-auth/README.md

```diff
@@ -46,20 +46,29 @@ Then, in your medusa config plugins collection you can add the following configu
     options: {
         // Enable google OAuth 2
         google: {
+            // ----------- REQUIRED -----------
             clientID: "__YOUR_CLIENT_ID__",
             clientSecret: "__YOUR_CLIENT_SECRET__",
+            
             // Enable google OAuth 2 for the admin domain
             admin: {
+                // ----------- REQUIRED -----------
                 callbackUrl:`${process.env.BACKEND_URL}/admin/auth/google/cb`, 
                 failureRedirect: `${process.env.ADMIN_URL}/login`,
                 successRedirect: `${process.env.ADMIN_URL}/`,
+                
+                // ----------- OPTIONAL -----------
```

Comment:
Could you add the authPath and authPathCallback default in the different section :D